sdk/identity/Azure.Identity/README.md

```diff
@@ -188,40 +188,40 @@ Not all credentials require this configuration. Credentials which authenticate t
 
 ### Authenticate Azure-hosted applications
 
-|Credential | Usage
-|-|-
-|[`DefaultAzureCredential`][ref_DefaultAzureCredential]|Provides a simplified authentication experience to quickly start developing applications run in Azure.
-|[`ChainedTokenCredential`][ref_ChainedTokenCredential]|Allows users to define custom authentication flows composing multiple credentials.
-|[`ManagedIdentityCredential`][ref_ManagedIdentityCredential]|Authenticates the managed identity of an Azure resource.
-|[`EnvironmentCredential`][ref_EnvironmentCredential]|Authenticates a service principal or user via credential information specified in environment variables.
+| Credential                                                   | Usage                                                                                                    | Built-in token caching? |
+|--------------------------------------------------------------|----------------------------------------------------------------------------------------------------------|-------------------------|
+| [`DefaultAzureCredential`][ref_DefaultAzureCredential]       | Provides a simplified authentication experience to quickly start developing applications run in Azure.   | N/A                     |
+| [`ChainedTokenCredential`][ref_ChainedTokenCredential]       | Allows users to define custom authentication flows composing multiple credentials.                       | N/A                     |
```

Comment:
I think we should do something more than "N/A" here like "it depends" with a link to a footnote explaining that is just depends on which credential in the chain you end up using.